tutorials/plugins/editor/inspector_plugins.rst

```diff
@@ -143,20 +143,22 @@ specifically add :ref:`class_EditorProperty`-based controls.
 
     public partial class MyInspectorPlugin : EditorInspectorPlugin
     {
-        public override bool _CanHandle(Variant @object)
+        public override bool _CanHandle(GodotObject @object)
         {
             // We support all objects in this example.
             return true;
         }
 
-        public override bool _ParseProperty(GodotObject @object, int type, string name, int hintType, string hintString, int usageFlags, bool wide)
+        public override bool _ParseProperty(GodotObject @object, Variant.Type type, 
+            string name, PropertyHint hintType, string hintString, 
+            PropertyUsageFlags usageFlags, bool wide)
```

Comment:
I don't think we usually split long method signature lines in the documentation, but I think it's fine to start doing it. I'm not sure what should be the maximum character length though.

Reply:
No worries either way - I was asked to break lines that were causing horizontal scroll (in both C# and GDScript) on an earlier PR, so I assumed that was the preferred approach.

The [C# style guide](https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_style_guide.html#general-guidelines) suggests 100 characters, but I had to break this at around 90 to avoid scroll.